trieste/acquisition/rule.py

```diff
@@ -62,12 +61,12 @@ def acquire(
         by the acquisition rule.
 
         **Type hints:**
-          - The global search space must be a :class:`~trieste.space.SearchSpace`. The exact type
-            of :class:`~trieste.space.SearchSpace` depends on the specific
-            :class:`AcquisitionRule`.
-          - Each :class:`AcquisitionRule` must define the type of its corresponding acquisition
-            state (if the rule is stateless, this type can be `None`). The ``state`` passed
-            to this method, and the state returned, must both be of that type.
+        - The global search space must be a :class:`~trieste.space.SearchSpace`. The exact type
+          of :class:`~trieste.space.SearchSpace` depends on the specific
+          :class:`AcquisitionRule`.
+        - Each :class:`AcquisitionRule` must define the type of its corresponding acquisition
+          state (if the rule is stateless, this type can be `None`). The ``state`` passed
+          to this method, and the state returned, must both be of that type.
```

Comment:
now you're going the other way

Reply:
eh?

Reply:
you seem to like (de-)indenting things

Reply:
it's good to keep PRs really focussed. This helps reviewers see what's changed and why, and also helps when rebasing on new changes, where conflicts are either avoided or are easier to fix

Reply:
I dont know why this is happening!